packages/dso-toolkit/components/Voorbeelden/20-Toepassingen/60-Helpcentrum/Startpagina.njk

```diff
@@ -0,0 +1,107 @@
+{% render '@header', {menu: menu, breadcrumbs: []} -%}
+<main>
+  <div class="row dso-banner dso-banner-implementation-specific-image" style="background-image: url('../../dummy/images/hands-on-trackpad.jpg')">
+    <!-- @Developers: Voeg dus geen inline styling toe maar genereer de betreffende styling in een class, zoals bv. `.dso-banner-implementation-specific image`. Inline styling doet het niet goed bij de Webrichtlijnen -->
+    <div class="col-lg-6 col-sm-8">
+      <dso-highlight-box white>
+        <div class="dso-rich-content">
+          <h1>Helpcentrum</h1>
+          <p>Heeft u een vraag over de werking van de Vergunningcheck of over geldende regelgeving? Wij hebben de veelgestelde vragen per thema voor u op een rij gezet.</p>
+        </div>
+      </dso-highlight-box>
+    </div>
+  </div>
+  <h2>Waarmee kunnen we u helpen?</h2>
+
+  <div class="row">
+    <div class="col-md-8">
+      {% render '@search-bar', search %}
+    </div>
+  </div>
+
+  <hr>
+
+  <h2>Alle onderwerpen bekijken</h2>
+
+  <div class="row">
+    <div class="col-md-4">
+      <h3>U wilt weten of u werkzaamheden mag uitvoeren</h3>
+      <nav>
+        {% render '@link-list', links1 %}
+      </nav>
+    </div>
+
+    <div class="col-md-4">
+      <h3>U wilt direct een aanvrag of melding indienen</h3>
+      <nav>
+        {% render '@link-list', links2 %}
+      </nav>
+    </div>
+
+    <div class="col-md-4">
+      <h3>U wilt weten welke plannen de Overheid heeft</h3>
+      <nav>
+        {% render '@link-list', links3 %}
+      </nav>
+    </div>
+  </div>
+
+  <div class="row">
+    <div class="col-md-4">
+      <h3>U wilt weten hoe het Omgevingsloket werkt</h3>
+      <nav>
+        {% render '@link-list', links4 %}
+      </nav>
+    </div>
+
+    <div class="col-md-4">
+      <h3>U wilt direct een aanvrag of melding indienen</h3>
+      <nav>
+        {% render '@link-list', links5 %}
+      </nav>
+    </div>
+  </div>
+
+  <hr>
+
+  <h2>Overige vragen</h2>
+
+  <div class="row dso-equal-heights">
+    <div class="col-md-6">
+      <dso-highlight-box white drop-shadow>
+        <div class="dso-rich-content">
+          <div class="row">
+            <div class="col-md-4">
+              <img src="../../dummy/images/sneeuwpop.png" class="img-circle" alt="Afbeelding van een sneeuwpop">
```

Comment:
waarom hier niet een {% render %} gebruiken voor de img? Dat doe je in Artikel.njk wel

Reply:
aangepast